test/components/form/lenra_form_checkbox_test.dart

```diff
@@ -0,0 +1,86 @@
+import 'package:flutter/material.dart';
+import 'package:lenra_components/component/lenra_button.dart';
+import 'package:lenra_components/component/lenra_checkbox.dart';
+import 'package:lenra_ui_runner/components/events/data/value_data.dart';
+import 'package:lenra_ui_runner/components/events/event.dart';
+import 'package:lenra_ui_runner/lenra_widget.dart';
+import 'package:lenra_ui_runner/widget_model.dart';
+import 'package:provider/provider.dart';
+import '../../test_helper.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('LenraForm should work properly', (WidgetTester tester) async {
```

Comment:
Create more test for the Checkbox and make them shorter too.
One test = one feature. create a dead simple UI with only the form and the checkbox to test each feature on a different test.

You can create helper function to simplify the creation of the boilerplate for each test